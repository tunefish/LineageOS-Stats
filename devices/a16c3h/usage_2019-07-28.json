{
  "13.0-20170119-unofficial-a16c3h": 4,
  "13.0-20170207-unofficial-a16c3h": 1,
  "13.0-20170207-unofficial-konstakang-a16c3h": 1,
  "13.0-20170207-unofficial-konstakang-y560": 1129,
  "13.0-20170227-unofficial-a16c3h": 6,
  "13.0-20170902-unofficial": 1,
  "13.0-20170902-unofficial-jalebi": 127,
  "13.0-20171028-nightly-a16c3h": 82,
  "14.1-20180221-a16c3h-extended": 8,
  "14.1-20180907-microg-a16c3h": 4,
  "14.1-20190105-nightly-a16c3h": 60,
  "15.1-20180718-go-brutality-a16c3h-unofficial": 490,
  "a16c3h-userdebug 7.1.2 njh47f 16f36b7273 release-keys": 4,
  "a16c3h-userdebug 7.1.2 njh47f 91a96bc9b4 release-keys": 12,
  "cm-boldxs.30": 1,
  "lineage-20170313-unofficial-a16c3h": 2,
  "nuclearom-20182512-unofficial": 1
}