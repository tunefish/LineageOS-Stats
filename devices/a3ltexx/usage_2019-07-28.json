{
  "14.1-20170204-unofficial-a3ltexx": 85,
  "14.1-20170330-unofficial-a3ltexx": 33,
  "14.1-20170525-official-a3ltexx": 1,
  "14.1-20170525-unofficial-a3ltexx": 1535,
  "14.2": 1
}