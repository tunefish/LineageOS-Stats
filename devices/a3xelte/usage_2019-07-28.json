{
  "15.1-20190125-unofficial-a3xelte": 36,
  "15.1-20190212-unofficial-a3xelte": 32,
  "15.1-20190310-unofficial-a3xelte": 38,
  "15.1-20190408-unofficial-a3xelte": 193,
  "15.1-20190412-dnjl-a3xelte": 1,
  "15.1-20190515-dnjl-a3xelte": 1,
  "15.1-20190516-unofficial-a3xelte": 189,
  "15.1-20190623-unofficial-a3xelte": 81,
  "15.1-20190709-unofficial-a3xelte": 2,
  "16.0-20190201-unofficial-a3xelte": 1,
  "16.0-20190205-unofficial-a3xelte": 4,
  "16.0-20190320-unofficial-a3xelte": 2,
  "16.0-20190326-unofficial-a3xelte": 23,
  "16.0-20190402-unofficial-a3xelte": 2,
  "16.0-20190405-unofficial-a3xelte": 4,
  "16.0-20190407-unofficial-a3xelte": 4,
  "16.0-20190408-unofficial-a3xelte": 160,
  "16.0-20190504-unofficial-a3xelte": 2,
  "16.0-20190511-unofficial-a3xelte": 1,
  "16.0-20190517-unofficial-a3xelte": 33,
  "16.0-20190521-unofficial-a3xelte": 29,
  "16.0-20190523-unofficial-a3xelte": 35,
  "16.0-20190525-unofficial-a3xelte": 1,
  "16.0-20190526-unofficial-a3xelte": 153,
  "16.0-20190625-unofficial-a3xelte": 99,
  "16.0-20190707-unofficial-a3xelte": 100,
  "16.0-20190711-unofficial-a3xelte": 1,
  "nrd90m.a310fxxu3cqe6": 1
}