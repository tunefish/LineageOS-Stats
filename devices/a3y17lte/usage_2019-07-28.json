{
  "14.1-20180205-unofficial-a3y17lte": 6,
  "14.1-20180614-unofficial-j5xnlte": 6084,
  "15.1-20180211-unofficial-a3y17lte": 2,
  "15.1-20180306-unofficial-a3y17lte": 5,
  "15.1-20180517-unofficial-a3y17lte": 17,
  "15.1-20180821-unofficial-a3y17lte": 29,
  "15.1-20180826-unofficial-a3y17lte": 1,
  "15.1-20181205-unofficial-a3y17lte": 451,
  "16.0-20181215-unofficial-a3y17lte": 105,
  "16.0-20190206-unofficial-a3y17lte": 1,
  "16.0-20190307-unofficial-a3y17lte": 59,
  "16.0-20190318-unofficial-a3y17lte": 1,
  "16.0-20190330-unofficial-a3y17lte": 45,
  "16.0-20190408-unofficial-a3y17lte": 1,
  "16.0-20190409-unofficial-a3y17lte": 3,
  "16.0-20190410-unofficial-a3y17lte": 15,
  "16.0-20190411-unofficial-a3y17lte": 2,
  "16.0-20190412-unofficial-a3y17lte": 37,
  "16.0-20190416-unofficial-a3y17lte": 48,
  "16.0-20190419-unofficial-a3y17lte": 147,
  "16.0-20190424-unofficial-a3y17lte": 41,
  "16.0-20190426-unofficial-a3y17lte": 242,
  "16.0-20190430-unofficial-a3y17lte": 74,
  "16.0-20190507-unofficial-a3y17lte": 35,
  "16.0-20190510-unofficial-a3y17lte": 111,
  "16.0-20190517-unofficial-a3y17lte": 39,
  "16.0-20190518-unofficial-a3y17lte": 43,
  "16.0-20190519-unofficial-a3y17lte": 18,
  "16.0-20190522-unofficial-a3y17lte": 188,
  "16.0-20190529-unofficial-a3y17lte": 221,
  "16.0-20190611-unofficial-a3y17lte": 227,
  "16.0-20190612-unofficial-a3y17lte": 1,
  "16.0-20190623-unofficial-a3y17lte": 370,
  "16.0-20190626-unofficial-a3y17lte": 13
}