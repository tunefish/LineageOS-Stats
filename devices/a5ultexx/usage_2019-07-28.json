{
  "14.1-20170114-unofficial-a5ultexx": 12,
  "14.1-20170216-unofficial-a5ultexx": 112,
  "14.1-20170817-unofficial-a5ultexx": 73,
  "14.1-20170822-unofficial-a5ultexx": 13,
  "14.1-20170906-unofficial-a5ultexx": 106,
  "14.1-20171231_020714-unofficial-a5ultexx": 52,
  "14.1-20180820-unofficial-a5ultexx": 30,
  "14.1-20180825-unofficial-a5ultexx": 2,
  "14.1-20180826-unofficial-a5ultexx": 1,
  "14.1-20180829-unofficial-a5ultexx": 2,
  "14.1-20180830-unofficial-a5ultexx": 3,
  "14.1-20181114-unofficial-a5ultexx": 2,
  "14.1-20190315-unofficial-a5ultexx": 1,
  "14.1-20190502-unofficial-a5ultexx": 1,
  "14.1-20190504-unofficial-a5ultexx": 6,
  "14.1-20190609-unofficial-a5ultexx": 2,
  "14.1-20190610-unofficial-a5ultexx": 4,
  "14.1-20190613-unofficial-a5ultexx": 2,
  "14.1-20190623-unofficial-a5ultexx": 1,
  "14.1-20190705-unofficial-a5ultexx": 2,
  "14.1-20190706-unofficial-a5ultexx": 2,
  "14.1-20190708-unofficial-a5ultexx": 1,
  "15.1-20180210-unofficial-a5ultexx": 181,
  "16.0-20181015-unofficial-a5ultexx": 1,
  "16.0-20181102-unofficial-a5ultexx": 12,
  "16.0-20190129-unofficial-a5ultexx": 2,
  "16.0-20190220-unofficial-a5ultexx": 4,
  "16.0-20190225-unofficial-a5ultexx": 34,
  "16.0-20190328-unofficial-a5ultexx": 108,
  "16.0-20190427-unofficial-a5ultexx": 329
}