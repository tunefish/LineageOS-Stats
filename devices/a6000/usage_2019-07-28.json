{
  "14.1 unoffical": 1,
  "14.1-20170125-unofficial-a6000": 10,
  "14.1-20170205-unofficial-a6000": 36,
  "14.1-20170210-unofficial-a6000": 16,
  "14.1-20170225-unofficial-a6000": 161,
  "14.1-20170228-unofficial-a6000": 6,
  "14.1-20170403-unofficial-a6000": 60,
  "14.1-20170422-nightly-a6000": 12,
  "14.1-20170514-unofficial-a6000": 7,
  "14.1-20170607-unofficial-a6000": 7,
  "14.1-20170614-unofficial-a6000": 10,
  "14.1-20170706-unofficial-a6000": 46,
  "14.1-20170706-unofficial-dev_harsh1998-a6000": 3,
  "14.1-20170721-unofficial-a6000": 2,
  "14.1-20170731-unofficial-hyperteam-a6000": 10,
  "14.1-20170813-unofficial-a6000": 8,
  "14.1-20170831-unofficial-a6000": 190,
  "14.1-20180108-official-a6000": 1,
  "14.1-20180108-unofficial-a6000": 4,
  "14.1-unofficial": 1,
  "15.1-20180210-hyper-a6000": 21,
  "15.1-20180210-hyper-rendang": 8,
  "15.1-20180216-hyper-a6000": 116,
  "15.1-20180219-hyper-a6000": 23,
  "15.1-20180228-unofficial-a6000": 1,
  "15.1-20180308-hyper-a6000": 21,
  "15.1-20180413-hyper-a6000": 70,
  "15.1-20180514-unofficial-a6000": 2,
  "15.1-20180604-hyper-a6000": 2,
  "15.1-20180611-hyper-a6000": 16,
  "15.1-20180708-unofficial-a6000": 1,
  "15.1-20180720-hyper-a6000": 6,
  "15.1-20180720-unofficial-a6000": 11,
  "15.1-20180724-hyper-a6000": 69,
  "15.1-20180807-hyper-stable-a6000": 552,
  "15.1-20180807-hyper-stable-rendang": 15,
  "15.1-20190220-unofficial-a6000": 1,
  "15.1-20190504-unofficial-a6000": 2,
  "15.1-20190505-unofficial-a6000": 4,
  "15.1-20190709-unofficial-a6000": 1,
  "16.0-20190301-unofficial-a6000": 45,
  "16.0-20190302-unofficial-a6000": 116,
  "16.0-20190305-unofficial-a6000": 14,
  "16.0-20190309-hyper-a6000": 34,
  "16.0-20190316-hyper-a6000": 184,
  "16.0-20190316-hyper-a6010": 2,
  "16.0-20190406-unofficial-a6000": 8,
  "16.0-20190407-unofficial-a6000": 71,
  "16.0-20190417-unofficial-a6000": 122,
  "16.0-20190417-unofficial-andromaxq": 1,
  "16.0-20190508-unofficial-a6000": 10,
  "16.0-20190509-unofficial-a6000": 201,
  "16.0-20190610-unofficial-a6000": 225,
  "16.0-20190610-unofficial-a6010": 19,
  "a5000_s119_150820_8g_row": 2,
  "nuclearom-v-unofficial[7.1.2]--20171117-2103": 58
}