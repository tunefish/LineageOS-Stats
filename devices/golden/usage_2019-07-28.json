{
  "13.0-20170118-unofficial-golden": 2,
  "13.0-20170121-unofficial-golden": 7,
  "13.0-20170129-unofficial-golden": 102,
  "13.0-20170224-unofficial-golden": 5,
  "13.0-20170302-unofficial-golden": 19,
  "13.0-20170310-unofficial-golden": 102,
  "13.0-20170328-unofficial-golden": 796,
  "13.0-20170420-unofficial-golden": 19,
  "13.0-20170905-unofficial-glee": 1,
  "14.1-20170122-unofficial-golden": 196,
  "14.1-20170318-unofficial-golden": 4,
  "14.1-20170402-unofficial-golden": 11,
  "14.1-20170415-unofficial-golden": 3,
  "14.1-20170419-unofficial-golden": 11,
  "14.1-20170503-unofficial-golden": 3,
  "14.1-20170507-unofficial-golden": 33,
  "14.1-20170516-unofficial-golden": 1,
  "14.1-20170522-unofficial-golden": 6,
  "14.1-20170609-unofficial-golden": 702,
  "14.1-20170610-unofficial-golden": 13,
  "14.1-20170628-unofficial-golden": 886,
  "14.1-20170724-unofficial-golden": 47,
  "14.1-20170817-unofficial-golden": 73,
  "14.1-20170920-unofficial-golden": 16,
  "14.1-20171011-unofficial-golden": 138,
  "14.1-20171215_192311-official-golden": 61,
  "14.1-20171215_192311-unofficial-golden": 1397,
  "14.1-20180109_002536-unofficial-golden": 391,
  "14.1-20181014-unofficial-golden": 2,
  "15.1-20181030-unofficial-golden": 6
}